src/rat/rat_basin.py

```diff
@@ -597,9 +597,10 @@ def rat_basin(config, rat_logger):
                 else: 
                     raise Exception('Step-9 was not run OR There was an error in creating reservoir shapefile using spatial join for this basin from the global reservoir vector file.')
             # Get Sarea
+            filt_options = config['GEE']['bot_filter']
```

Comment:
filt_options = config['GEE'].get('bot_filter') so that even if bot_filter is not there in the config file, it will return None.

Reply:
Fixed